modules/server/src/main/java/org/jboss/wsf/stack/cxf/security/authentication/SubjectCreatingInterceptor.java

```diff
@@ -185,7 +171,7 @@ protected void setSubject(String name, String password, boolean isDigest, String
    private boolean checkUserPrincipal(Set<Principal> principals, String name)
    {
       for (Principal p : principals) {
-         if (groupClass == null || !groupClass.isInstance(p)) {
+         if (!(p instanceof Group)) {
```

Comment:
Same for https://issues.redhat.com/browse/JBWS-4133.

Reply:
changed